src/chigame/games/views.py

```diff
@@ -91,6 +91,26 @@ def test_func(self):
         return self.request.user.is_staff
 
 
+def search_results(request):
+    query = request.GET.get("query")
+
+    """
+    The Q object is an object used to encapsulate a collection of keyword
+    arguments that can be combined with logical operators (&, |, ~) which
+    allows for more advanced searches. More info can be found here at
+    https://docs.djangoproject.com/en/4.2/topics/db/queries/#complex-lookups-with-q-objects
+    """
+    object_list = Game.objects.filter(
+        Q(name__icontains=query)
+        | Q(categories__name__icontains=query)
+        | Q(people__name__icontains=query)
+        | Q(publishers__name__icontains=query)
```

Comment:
Appreciate the syntax here to separate each query component onto a new line with the operation at the front.

Both of these decisions drastically improve readability and ease of extension.